README.md

```diff
@@ -1 +1,16 @@
-# hb-multipage-site
```

Comment:
Try to add the original README with the correct format. For this check the original file and copy the code itself.

Reply:
Hi i just did it